lib/cli/versions.json

```diff
@@ -0,0 +1,16 @@
+{
+  "@storybook/angular": "6.0.0-alpha.43",
```

Comment:
will these versions always be the same and match with all other @storybook/*?

Reply:
yes, it's auto-generated in a prepare script!